javascript/javascript1/week3/week3.js

```diff
@@ -0,0 +1,170 @@
+//Item array removal
+let names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+for (i = 0; i < names.length; i++) {
+  if (names[i] === nameToRemove) {
+    names.splice(i, 1);
+  }
+}
+console.log(names);
+
+//When will we be there
+
+function getHours(speed, destinationDistance) {
+  const hours = destinationDistance / speed;
+  function formatTime(hours) {
```

Comment:
It can quickly get confusing nesting functions inside one another. I'd recommend you separating them instead.